lib/vtls/wolfssl.c

```diff
@@ -232,7 +232,6 @@ static const struct group_name_map gnm[] = {
 static int wolfssl_bio_cf_create(WOLFSSL_BIO *bio)
 {
   wolfSSL_BIO_set_shutdown(bio, 1);
-  wolfSSL_BIO_set_init(bio, 1);
```

Comment:
Does wolfSSL need any BIO initialization then? This code was probably copied from [ossl_bio_cf_create](https://github.com/curl/curl/blob/curl-8_6_0/lib/vtls/openssl.c#L673-L682). I can't find wolfSSL_BIO_set_init documented anywhere in wolfSSL. Nor shutdown for that matter.

@icing since the openssl and wolfssl create function is only used on bio creation why is there a shutdown call?

Reply:
wolfSSL_BIO_set_shutdown() at least does something - it stores that it has been shutdown...